docs/BEMAN_PROCESS.md

```diff
@@ -0,0 +1,48 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# The Beman Process
+
+This document specifies stages and best practices for Beman Project libraries development. Its goal is to create consistency facilitating the evaluation of, and contribution to Beman libraries.
+
+## Introduction
+
+### Changing this document
+
+This is a living document that adapts to evolving best practices and community
+needs. To make changes:
+
+1. Create a [discourse topic](https://discourse.boost.org) detailing the change
+   and how it aligns with the core principles.
+2. After some community discussion, create a PR with the actual change on
+   [GitHub](https://github.com/bemanproject/beman) with a *leads question*
```

Comment:
```suggestion
   [GitHub](https://github.com/bemanproject/beman) with a *leads question*
```
do we have this label?

Reply:
No, we didn't. I created a new label `!!! Beman leads required !!!`. Pushed the change in the PR and applied the label on current PR.

Thanks!